sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/models/EncryptionDatabaseResponse.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.EncryptionCosmosAsyncClient;
+
+public class EncryptionDatabaseResponse {
```

Comment:
ditto, why can't use the existing `CosmosDatabaseResponse`? why do we need additional public type?